README.md

```diff
@@ -42,14 +42,18 @@ Chat with us on [Matrix][chat-url].
 
 ## Getting Started
 
+The `config.yaml.example` file provides a working configuration for Threat Bus
+with all existing application plugins enabled together with the RabbitMQ
+backbone.
+
 The following example shows how to connect [MISP][misp], [Zeek][zeek] via
 Threat Bus. There are more integrations available, so make sure to check out all
 [Threat Bus projects on PyPI](https://pypi.org/search/?q=threatbus).
 
 *Start Threat Bus*
 
 ```sh
-venv/bin/threatbus -c config.yaml
+venv/bin/threatbus -c config.yaml.example
```

Comment:
Is this the procedure users should go through? I'd imagine it's typically (1) copy the example conf, and then (2) run with my own conf.

Reply:
I changed it such that the file is renamed first.